src/lib/parsers/sol-json.c

```diff
@@ -512,23 +512,15 @@ sol_json_escape_string(const char *str, char *buf, size_t len)
     return out;
 }
 
-SOL_API int
-sol_json_double_to_str(const double value, char *buf, size_t len)
+static void
+double_to_str(char *buf, size_t len)
```

Comment:
even if it's internal, name matters.

As for 71... then check the required size, you can round up to make sure.